build/pmm-client-docker/entrypoint.py

```diff
@@ -46,7 +48,7 @@ def main():
         print('Starting pmm-agent setup ...', file=sys.stderr)
         status = subprocess.call(['pmm-agent', 'setup'])
         print('pmm-agent setup exited with {}.'.format(status), file=sys.stderr)
-        if status != 0:
+        if status != 0 and not PMM_AGENT_SIDECAR:
```

Comment:
If we failed to do the setup, do we continue to work too? It looks slightly strange to me. We don't run setup anymore in this script and it means that we are break and will not be able to fix anything later. Why we keep working?